packages/synapse-react-client/src/components/EntityFinder/EntityFinder.tsx

```diff
@@ -188,114 +180,124 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
     }
   }, [accessToken, searchTerms, handleError])
 
+  const handleTabChange = (
+    _event: React.SyntheticEvent,
+    newValue: EntityFinderTab,
+  ) => {
+    setSelectedEntitiesSnapshot(selectedEntities)
+    setCurrentTab(newValue)
+  }
+
   const mainPanelClass = searchActive
     ? 'MainPanelSearch'
     : treeOnly
     ? 'MainPanelSinglePane'
     : 'MainPanelDualPane'
 
-  let searchButtonText
-  const tableArray = [
-    'table',
-    'entityview',
-    'materializedview',
-    'submissionview',
-    'dataset',
-    'datasetcollection',
-  ]
-  if (selectableTypes.length === 1) {
-    searchButtonText = `Search for ${pluralize(
-      entityTypeToFriendlyName(selectableTypes[0]),
-    )}`
-  } else if (selectableTypes.every(element => tableArray.includes(element))) {
-    searchButtonText = 'Search for Tables'
-  } else {
-    searchButtonText = 'Search all of Synapse'
+  const entityDetailsListProps: EntityDetailsListProps = {
+    configuration: configFromTreeView,
+    versionSelection: versionSelection,
+    selected: selectedEntities,
+    isIdSelected: isIdSelected,
+    isSelectable: isSelectable,
+    visibleTypes: selectableAndVisibleTypesInList,
+    selectableTypes: selectableTypes,
+    selectColumnType: selectMultiple ? 'checkbox' : 'none',
+    toggleSelection: toggleSelection,
+    enableSelectAll: selectMultiple,
+    setCurrentContainer: setCurrentContainer,
   }
 
   return (
     <SynapseErrorBoundary>
       <div className="EntityFinder">
         <Box
           display="flex"
-          justifyContent="flex-end"
-          alignItems="stretch"
-          pb={1}
+          justifyContent="space-between"
+          mb={2.5}
+          flexWrap={'wrap'}
+          rowGap={2}
+          columnGap={4}
+          sx={{
+            flexWrap: { xs: 'wrap', md: 'nowrap' },
+          }}
```

Comment:
Just kinda tweaked this until it felt good

[Screencast from 11-13-2024 11:51:03 AM.webm](https://github.com/user-attachments/assets/93de7a0d-7059-457f-a897-0a2d50ad059b)

Reply:
I think this is fine, but there may be some confusion on the scope of this search box (users may think this is searching the selected entities table, for example).
The real mobile responsive issue is the browse UX, which I think is out of scope for this ticket

Reply:
Great point